### ISSUE TITLE
Port conflicts!

### ISSUE THREAD

**Issue by slimsag** [sourcegraph/sourcegraph#52398]
The plan - do the lightweight work to make this less of an issue:

1. Disable debug server running on port :6060 (we don't need it)
2. Set all ports to really high numbers if we can, to avoid conflicts best we can.

**Comment by taylorsperry** [comment]
William is working on this, so assigning him and moving to "In Progress."

**Comment by huyz** [comment]
Yes, please do not use ports 49000 (Portainer) or 3080 (developers often use `*080` ports for local dev purposes). In the past 2 months, I've run into port conflicts, which I've never run into before. And both times it's because of the Cody App. I really don't understand why.  There are so many ports out there. Why pick `080` or `000`. Random numbers would work best :)
